pages/index.vue

```diff
@@ -1,10 +1,10 @@
 <template>
   <div class="page">
-    <pw-section class="blue" label="Request" ref="request">
+    <pw-section class="blue" label="Import" ref="import">
       <button id="show-modal" @click="showModal = true">IMPORT</button>
```

Comment:
Can you change this from 'IMPORT' to 'Import'?
Other than that, everything's perfect.

Reply:
Done.